spec/spec_helper.rb

```diff
@@ -21,6 +19,8 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+load Rails.root + "db/seeds_test.rb"
+
```

Comment:
I don't think any of these seeds are necessary at this point. IIRC they were created right after I hacked in roles, but prior to factories.